.github/workflows/linux-eic-shell.yml

```diff
@@ -289,16 +298,19 @@ jobs:
         setup: install/setup.sh
         run: |
           mkdir -p doc
-          npdet_info dump ${DETECTOR_PATH}/epic.xml | tee doc/constants.out
+          npdet_info dump ${DETECTOR_PATH}/${{matrix.detector_config}}.xml | tee doc/${{matrix.detector_config}}_constants.out
     - uses: actions/upload-artifact@v3
       with:
-        name: constants.out
-        path: doc/constants.out
+        name: constants
+        path: doc/${{matrix.detector_config}}_constants.out
         if-no-files-found: error
 
   dump-parameter-table:
     runs-on: ubuntu-latest
     needs: build-gcc
+    strategy:
+      matrix:
+        detector_config: [epic_arches, epic_brycecanyon]
```

Comment:
and here?

Reply:
For this one I'm pretty sure it wouldn't work because e.g. drich_only doesn't define a lot of constants that are used in the table template.